.github/workflows/push-dev-environment.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           NAMESPACE: lifebank
           STAGE: dev
           APP_NAME: lifebank
-          INGRESS_HOST: lifebank.io
+          INGRESS_HOST: lifebank.eosio.cr
```

Comment:
Is this change ok?  or there should be lifebank.io and lifebank.eosio.cr separated?

Reply:
lifebank.io